File: bms/go/go-gt/Matrix_Print/Matrix_Print.go
package test

import (
	"fmt"
)

var ()

const ()

func (m *Matrix) Print() {
	var i, j int64
	for i = 0; i < m.N; i++ {
		for j = 0; j < m.N; j++ {
			fmt.Printf("%d ", m.Get(i, j))
		}
		fmt.Print("\n")
	}
}

func (m Matrix) Get(i int64, j int64) int64 {
	return m.A[i*m.N+j]
}

type Matrix struct {
	N int64
	A []int64
}

File: translations_only/success/go/go-gt/Matrix_Print/claude2/BaseRepair/Matrix_Print.rs

use std::fmt;

struct Matrix {
    n: usize,
    a: Vec<i64>,
}

fn print(m: &Matrix) {
    for i in 0..m.n {
        for j in 0..m.n {
            print!("{} ", get(m, i, j));
        }
        println!("");
    }
}

fn get(m: &Matrix, i: usize, j: usize) -> i64 {
    m.a[i * m.n + j] 
}

